collectors/security/coverity/src/main/java/com/capitalone/dashboard/model/CoverityCollector.java

```diff
@@ -0,0 +1,25 @@
+package com.capitalone.dashboard.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CoverityCollector extends Collector {
+	
+	public static final String NICE_NAME = "Coverity";
+	
+    private List<String> coverityServers = new ArrayList<>();
+
+    public List<String> getCoverityServers() {
+        return coverityServers;
+    }
+
+    public static CoverityCollector prototype(List<String> servers) {
```

Comment:
@fidelcoria-aa  How is coverity project uniquely identified?  project is mapped to collector-item , but there are options/unique options specified. Refer SonarCollector.java

Reply:
Mmm... Do you mean CoverityProject.java?

Reply:
@fidelcoria-aa  Nope, take sonar collector 
{
    "_id" : ObjectId("56cb5bde7fab7c548551410e"),
    "_class" : "com.capitalone.dashboard.model.Collector",
    "name" : "Sonar",
    "collectorType" : "CodeQuality",
    "enabled" : true,
    "online" : true,
    "errors" : [],
    "uniqueFields" : {
        "projectName" : "",
        "instanceUrl" : ""
    },
    "allFields" : {
        "projectName" : "",
        "projectId" : "",
        "instanceUrl" : ""
    },
    "lastExecuted" : NumberLong(1551810772348),
    "searchFields" : [ 
        "options.projectName", 
        "niceName"
    ],
    "properties" : {}
}

can you send collector document from Mongo for Coverity collector. I am looking of block in Coverity collector 

"uniqueFields" : {
        "projectName" : "",
        "instanceUrl" : ""
    },
    "allFields" : {
        "projectName" : "",
        "projectId" : "",
        "instanceUrl" : ""
    },

Reply:
"uniqueFields" : {},
    "allFields" : {},
Oooooh :open_mouth:.  I think I understand what you are saying. `uniqueFields` and `allFields` contain the fields of the collector-item that identify a unique item and all fields, respectively. 

Is this just for documentation purposes? I assume the values in these objects are meant for documentation. I don't see these values used anywhere. Or should I use them?